### PR TITLE
Refactor/lendingpool size

### PR DESCRIPTION
**Pull request by j-vp**

### CODE REVIEW

**Comment by j-vp** [comment]
Lendingpool is down to 24.241kb,  0.335kb over at 20 optimizations. Will check for max possible optimizations

**Comment by j-vp** [comment]
also good to go at 200 runs

File: foundry.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ out = 'out'
 test = 'test'
 libs = ['lib']
 optimizer = true
-optimizer_runs = 200
+optimizer_runs = 20
```
Comment (Thomas-Smets) [inline foundry.toml:7]: We keep it at 20 or back to 200?

**Review by Thomas-Smets — APPROVED** [review]
gtg

**Pull request merged**